WalletWasabi/Helpers/LinuxStartupChecker.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Helpers
+{
+	public static class LinuxStartupChecker
```

Comment:
Also ideally, one would not need two classes `LinuxStartupHelper` and `LinuxStartupChecker`. Is there a reason for having two classes instead of just one?

Reply:
Yes, I needed to move business logic out of the `ViewModel`, the check is now in the `WalletWasabi.Gui` project, and from there i cannot reach the `Fluent Helpers` (circular reference). That's why I split apart the "Helper" and the "Checker" parts.

Reply:
Do we want the startup functionality in GUI project? I thought that all new development is only in Fluent.

Reply:
No, but the `Fluent.Desktop/Program.cs` creates and uses the `WW.Gui/Global.cs`.

Fluent.Desktop/Program.cs (155-160)
```
private static Global CreateGlobal(string dataDir, UiConfig uiConfig, Config config)
		{
			var walletManager = new WalletManager(config.Network, dataDir, new WalletDirectories(config.Network, dataDir));

			return new Global(dataDir, config, uiConfig, walletManager);
		}
```
where the `new Global(dataDir, config, uiConfig, walletManager)` refers to the Gui project's `Global.cs`

Reply:
I see thanks. 

Would it make sense to move your new classes to `WalletWasabi.Gui` project then so that once the project is removed, the files are moved to fluent project? Otherwise, those files will stay in base project forever which is imho not good.

Reply:
IMO having Fluent feature helpers in `WalletWasabi.Gui` project is a no-go. Especially if we plan to delete the Gui project in the future. One has to remember to move some specific files before the deletion and who knows when the deletion will happen.